assets/app/views/createRoute.html

```diff
@@ -0,0 +1,52 @@
+<div class="content">
+  <div class="container">
+    <div class="col-md-12">
+      <ol class="breadcrumb">
+        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
+        <li class="active"><strong>Add Route</strong></li>
+      </ol>
+      <alerts alerts="alerts"></alerts>
+      <h1>Add Route</h1>
+      <div ng-if="serviceName">
+        <span class="help-block">Create a route for service {{serviceName}}.</span>
+      </div>
+      <div ng-if="!serviceName">
+        <span class="help-block">Select a service and enter route details.</span>
```

Comment:
Hmm the externally part im not sure of, might scare people into thinking
they are making it totally public (like outside a firewall).  "Select a
service to expose"  or
"Select a service to route to" ?

On Tue, Nov 24, 2015 at 4:33 PM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/createRoute.html
> <https://github.com/openshift/origin/pull/5997#discussion_r45797884>:
>
> > @@ -0,0 +1,52 @@
> > +<div class="content">
> > +  <div class="container">
> > +    <div class="col-md-12">
> > +      <ol class="breadcrumb">
> > +        <li><a href="{{projectName | projectOverviewURL}}">{{(project | displayName) || projectName}}</a></li>
> > +        <li class="active"><strong>Add Route</strong></li>
> > +      </ol>
> > +      <alerts alerts="alerts"></alerts>
> > +      <h1>Add Route</h1>
> > +      <div ng-if="serviceName">
> > +        <span class="help-block">Create a route for service {{serviceName}}.</span>
> > +      </div>
> > +      <div ng-if="!serviceName">
> > +        <span class="help-block">Select a service and enter route details.</span>
>
> "Select a service to expose externally"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5997/files#r45797884>.
>